octobot_trading/enums.py

```diff
@@ -156,6 +156,14 @@ class TraderPositionType(enum.Enum):
     ISOLATED = "isolated"
 
 
+class PositionSubType(enum.Enum):
+    LINEAR = "linear"
+    INVERSE = "inverse"
+    SWAP = "swap"
+    OPTION = "option"
+    SUB_TYPE = "subType"
+    SETTLE_COIN = "settleCoin"
```

Comment:
Are they ccxt values?

Reply:
Oh yeah you're right its not ccxt. I'll move it to bybit